apps/dot-voting/contracts/DotVoting.sol

```diff
@@ -36,7 +36,14 @@ contract DotVoting is ADynamicForwarder, AragonApp {
     bytes32 constant public MODIFY_QUORUM = keccak256("MODIFY_QUORUM");
     bytes32 constant public MODIFY_CANDIDATE_SUPPORT = keccak256("MODIFY_CANDIDATE_SUPPORT");
     bytes32 constant public ADD_CANDIDATES_ROLE = keccak256("ADD_CANDIDATES_ROLE");
-    string constant QUORUM_SUPPORT_ERROR= "Global quorum must be greater or equal to candidate support";
+    string private constant QUORUM_SUPPORT_ERROR = "QUORUM_SUPPORT_ERROR";
+    string private constant VOTE_ID_RANGE = "VOTE_ID_RANGE";
+    string private constant CANDIDATE_RANGE_ERROR =  "CANDIDATE_RANGE_ERROR";
+    string private constant MIN_QUORUM_ISSUE = "MIN_QUORUM_ISSUE";
+    string private constant CAN_VOTE_ERROR = "CAN_VOTE_ERROR";
+    string private constant CAN_EXECUTE_ERROR = "CAN_EXECUTE_ERROR";
+    string private constant VOTE_LENGTH_ERROR = "VOTE_LENGTH_ERROR";
+    string private constant TOTAL_SUPPORT_ERROR = "TOTAL_SUPPORT_ERROR";
```

Comment:
Not a big deal if this doesn't happen, but I'd prefer we sorted these sorts of lists alphabetically. I think it would make it easier to find a relevant error code to use.

I'd also prefer we standardized on `ERROR` or `ISSUE` instead of mixing the two.

Looking up to the ROLE constants, it also looks like we do not use consistent naming with those. Would it make sense to prepend all of these constants with `ERROR_` or `ROLE_`, rather than putting it and the end or, in some cases, omitting it entirely?

Reply:
Note that I am not proposing we change the strings, only the constant names. The strings could potentially omit the words "error" and "role" altogether, such as:

```sol
string private constant ERROR_CANNOT_VOTE = "Voting Forbidden"
```

I think I probably like the "error-code" style more than stripped-down error messages, but I wanted to throw it out as a possibility.

Reply:
I am in favour of using better messages while keeping the same constants names, I suggest using the same methodology as A1 does for their contracts, it should not be really hard to be a bit more expressive without using long strings, I think a good measure could be targeting something like 30 chars long strings (based on the average length for these reasons on other apps/contracts. also from A1)

Reply:
I'm fine to sort alphabetically and prepend ERROR_ or ROLE_ but I think the error message should just be the name, it's strictly an identifier and when I see an error the first thing I do is just key into the code, so being able to search for the actual `CONST` and get to the specific require is just quicker for me.

Reply:
https://github.com/aragon/aragon-apps/blob/master/apps/survey/contracts/Survey.sol#L25 A1 just uses the methadology of switching ERROR_ for the app name. That's fine with me but it's pretty clear what contract the error comes from. I don't want like, long sentence errors to me that's just not useful.